internal/controllers/device_data_controller_test.go

```diff
@@ -727,3 +728,90 @@ func TestUserDevicesController_GetUserDevicesErrorCodeQueries(t *testing.T) {
 		test.TruncateTables(pdb.DBS().Writer.DB, t)
 	})
 }
+
+func TestUserDevicesController_ClearUserDeviceErrorCodeQuery(t *testing.T) {
+	mockDeps := createMockDependencies(t)
+	defer mockDeps.mockCtrl.Finish()
+
+	ctx := context.Background()
+	pdb, container := test.StartContainerDatabase(ctx, t, migrationsDirRelPath)
```

Comment:
We’ll have to find a way, at some point, to avoid starting this with every test. These are taking longer and longer to run.

Reply:
yeah, we need something like a beforeAll or maybe a way to put everything in a global variable and run it from there